packages/embed-grid/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <base href="./" />
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+
+    <script>
+      // Can't be an HTML meta redirect because we need to keep the search params
+      const url = new URL('../../iframe/widget/', document.baseURI);
```

Comment:
May as well just do `../widget` ...
```suggestion
      const url = new URL('../widget/', document.baseURI);
```

We could have env vars that you set when building to reference where the new iframe path should be, but this is fine... if you change the path, you'll need to rewrite the path in this index.html file.

Reply:
Ya I figured these were simple enough redirects that adding an env file just adds an unnecessary layer of pass through. Not any gain IMO when we're replacing a single use value with a single use variable instead and we're ultimately trying to remove these separate embed packages